src/main/java/org/jabref/model/groups/AutomaticGroup.java

```diff
@@ -2,7 +2,10 @@
 
 import java.util.Set;
 
+import javafx.collections.ObservableList;
```

Comment:
I agree that we don't want to have GUI technology in our model and logic classes. However, these observable lists and properties have nothing to do with the GUI-related part of JavaFX - you can use them in a command line application if you want. An observable list is just a list with some built-in event listeners, i.e. a different form of List combined with the google event stuff. Similar the properties in in `javafx.beans` are just fields with event handlers.

Thus I strongly vote for allowing `javafx.collections` and `beans` in model and logic.